### PR TITLE
Module 1 - Get Repository Information #1

### PR DESCRIPTION
**Pull request by jacobeicher**
Here are the changes on the pull request for module 1

I apologize for the branch change, but we need to preserve the general commit history of development for the grading process

### CODE REVIEW

File: .github/workflows/linter.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        python-version: 3.9
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install pylint
+        pip install pyGitHub
+    - name: Analysing the code with pylint
+      run: |
+        find . -type f -name "*.py" | xargs pylint --exit-zero
+
```
Comment (klfgit) [inline .github/workflows/linter.yaml:26]: add a newline.

**Review by klfgit — CHANGES_REQUESTED** [review]

File: .github/workflows/linter.yaml (hunk 1 above)
Comment (jacobeicher) [reply .github/workflows/linter.yaml:26]: A second new line to the end of the file, or where do you mean?

Comment (klfgit) [reply .github/workflows/linter.yaml:26]: github is indicating that the file did not end with a newline.

Comment (klfgit) [reply .github/workflows/linter.yaml:24]: @klfgit test

Comment (klfgit) [reply .github/workflows/linter.yaml:26]: Screen Shot 2021-11-18 at 8 40 49 AM

@jacobeicher  ^^

File: README.md [hunk 2]
```diff
@@ -1,2 +1,63 @@
...
+###### _**NOTE**: Tokens should be encrypted and only decrypted at runtime_
+
+## Testing with 'ansible-test'
+
+Testing has been made available using the [ansible-test](https://docs.ansible.com/ansible/latest/dev_guide/testing_integration.html). These tests include [unit](https://github.com/senior-design-21-22/ansible-collection-github/tree/repo-information-module/tests/unit), [sanity](https://github.com/senior-design-21-22/ansible-collection-github/tree/repo-information-module/unit/sanity), integration.
+
+The tests are runnable using the following commands:
+
+```bash
+ansible-test unit
```
Comment (klfgit) [inline README.md]: This should be `ansible-test units`, right?

**Review by klfgit — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -1,2 +1,63 @@
...
+### Repository Information (From organization)
+
+```
+  - name: "List GitHub repositories within an organization"
+      import_role:
+        name: ohioit.github.repository_info
+      vars:
+        github_token: "token"
+        org_name: "organization_name"
+```
```
Comment (klfgit) [inline README.md]: Add and example of how to use the module directly.

**Review by klfgit — CHANGES_REQUESTED** [review]

File: .github/workflows/linter.yaml (hunk 1 above)
Comment (jacobeicher) [reply .github/workflows/linter.yaml:26]: image

That's very strange, it looks like its there in the request and Its there on the branch

File: README.md (hunk 2 above)
Comment (Khounborinn) [reply README.md]: Yes, the changes are made and will be a part of the next pull request

File: README.md (hunk 3 above)
Comment (TylerZwolenik) [reply README.md]: This example has now been modified to work with the new collection format without roles.

**Pull request closed without merging**